repobuild/upload.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+
+me=$(basename $0)
+dir=$(readlink -f $(dirname $0))
+
+usage()
+{
+    echo "This script uploads files to an s3 backet."
+    echo "Usage example: $me -s /path/to/source/files/* -b my-s3-backet -t /path/in/the/backet [-e ./**/exclude_files] [-n '*no-cache-files*'] [-t my_tag=some_value] -acl-public | -h"
+    echo
+    echo "  -s | --source          : Source loacation of the packages, like 'linux/repo/artifacts/deb/'. The relative path from the current directory."
+    echo "  -b | --backet          : Target backet name."
```

Comment:
Ah I see you made the same mistake, consistently, everywhere )

Change them all to `bucket`

Reply:
Yes ) Fixed.